src/__test__/keyboard-zoom.test.ts

```diff
@@ -0,0 +1,184 @@
+// Mock the zig imports to avoid compilation issues in tests
```

Comment:
Remove this whole test file. Add an e2e test to `integration-tests/tests/camera.spec.ts` with name 'zoom/un-zoom with keyboard only'